types/event_bus.go

```diff
@@ -10,7 +10,7 @@ import (
 	"github.com/cometbft/cometbft/libs/service"
 )
 
-const defaultCapacity = 0
+const defaultCapacity = 1000
```

Comment:
I guess this is a part of other PR ;)

Reply:
yeah sorry was going to revert was just trying to figure out why the e2e was halting in CI but not locally